internal/pkg/get/get.go

```diff
@@ -0,0 +1,78 @@
+package details
+
+import (
+	"fmt"
+
+	"github.com/jbrekelmans/kube-compose/internal/pkg/k8smeta"
+	"github.com/jbrekelmans/kube-compose/pkg/config"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	clientV1 "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+type getRunner struct {
+	cfg              *config.Config
+	k8sClientset     *kubernetes.Clientset
+	k8sServiceClient clientV1.ServiceInterface
+}
+
+type KubeComposeServiceDetails struct {
```

Comment:
Just `ServiceDetails` without the `KubeCompose`

Reply:
Done